recipe/build.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 
+# Avoid Xcode
+if [[ $(uname) == Darwin ]]; then
+  PATH=${PREFIX}/bin/xc-avoidance:${PATH}
+fi
```

Comment:
Looks like we should be using `conan` on OSX also: https://github.com/qtwebkit/qtwebkit/wiki/Building-QtWebKit-on-macOS